common/scala/Dockerfile

```
@@ -14,8 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
-FROM adoptopenjdk/openjdk8:x86_64-alpine-jdk8u172-b11
+FROM adoptopenjdk/openjdk11:x86_64-alpine-jdk-11.0.3_7
```

Per suggestion on the dev list - this reduces performance degradation. On the tech interchange call we discussed making this a separate PR but I think based on the dev list exchange we can just accept this commit and move on.
```suggestion
FROM adoptopenjdk/openjdk11-openj9:x86_64-alpine-jdk-11.0.5_10_openj9-0.17.0
```